### PR TITLE
Support for Distilbert

### PR DESCRIPTION
**Pull request by andrelmfarias**
This PR adds support for Distilbert as mentioned in #197.

I will be also releasing a Distilbert trained on SQuAD 1.1 using Knowledge Distillation and `bert-large-uncased-whole-word-masking-finetuned-squad` as a teacher.

From my experiments, this version of Distilbert achieves 80.1% EM and 87.5% F1-score (vs. 81.2% EM and 88.6% F1-score for our version of BERT), while being much faster and lighter.

### CODE REVIEW

**Comment by yunhaizhu** [comment]
How did you train and dump "distilbert_qa.joblib"? it seems that many files for distilbert did not updated.

**Comment by andrelmfarias** [comment]
> How did you train and dump "distilbert_qa.joblib"? 

`distilbert_qa.joblib` is an instance of [`BertQA`](https://github.com/cdqa-suite/cdQA/blob/50e10443bc37b7bd3546465d37059260f29549f0/cdqa/reader/bertqa_sklearn.py#L999), which has a torch model corresponding to the fine-tuned Distilbert model. I trained this torch model it using the official Hugging Face's repo for distillation on SQuAD with the only difference that I replaced `bert-base-uncased` by `bert-large-uncased-whole-word-masking-finetuned-squad`.

> it seems that many files for distilbert did not updated.

Which files are you talking about?

**Comment by yunhaizhu** [comment]
> > How did you train and dump "distilbert_qa.joblib"?
> 
> `distilbert_qa.joblib` is an instance of [`BertQA`](https://github.com/cdqa-suite/cdQA/blob/50e10443bc37b7bd3546465d37059260f29549f0/cdqa/reader/bertqa_sklearn.py#L999), which has a torch model corresponding to the fine-tuned Distilbert model as attribute. I trained this torch model it using the official Hugging Face's repo for distillation on SQuAD with the only difference that I replaced `bert-base-uncased` by `bert-large-uncased-whole-word-masking-finetuned-squad`.

Thanks for the reply, Got it. 

> > it seems that many files for distilbert did not updated.
> 
> Which files are you talking about?

Oh, I checked the bertqa_sklearn.py, DistilBert Model  and corresponding classes did not use.
